### ISSUE TITLE
Emacs-style navigation does not work when replaceInputFocussed

### ISSUE THREAD

**Issue by ANorwell** [whitphx/vscode-emacs-mcx#480]
As of https://github.com/tuttieee/vscode-emacs-mcx/pull/466, regular emacs navigation commands do not apply when `replaceInputFocussed` (i.e. when the search and replace is visible and the cursor is in the replace section). However, emacs-style keybindings do not apply when this is focussed. Some examples:
- Ctrl-F opens the file dialog instead of moving forward a character
- Ctrl-A highlights all instead of moving to the start of the line.

I tried customizing hotkeys to get around this. For example:

```
      {
        "key": "ctrl+b",
        "command": "cursorLeft",
        "when": "editorFocus && findWidgetVisible && replaceInputFocussed",
      },
```

But, this does not work -- the default vscode behaviour stops happening, but the navigation doesn't happen either. Similarly for the emacs-mcx style commands such as `emacs-mcx.forwardChar`, if they are used as the `command` in the custom hotkey. It seems like the `cursorX` commands just don't work within the replace dialog.

Is there a way to get emacs-style hotkeys working within the replace input box? Why doesn't customizing hotkeys work properly here?

**Comment by tuttieee** [comment]
Are you using Windows?

* On search and replace widgets (and some UIs), VSCode keybindings do not work and OS' native shortcuts are applied. It's a problem of VSCode.
* On windows, Ctrl-F opens the file dialog and Ctrl-A selects all texts.
* As stated above, if you assign `emacs-mcx.forwardChar` to Ctrl-F, it does not work on the search and replace widget.

**Comment by tuttieee** [comment]
There has been the same issue but I have left it... #55 
The author of the issue suggested to disable all keybindings on the search and replace widget as a workaround because it's better that any keys do not work than that some keys work in unexpected ways.
To do the suggested workaround, please see the "workaround" section in #55

**Comment by whitphx** [comment]
The workaround suggested in #55 will be included in #755 and in the next release.
With that, Emacs-like cursor movements will be disabled on the find and replace widget on Windows.
